src/glomap/scene/image_pair.h

```diff
@@ -21,9 +21,6 @@ struct ImagePair : public colmap::TwoViewGeometry {
   // Second column is the index of the feature in the second image.
   Eigen::MatrixXi matches;
```

Comment:
Use FeatureMatches instead of Eigen::MatrixXi? Eigen::MatrixXi probably also has an inconvenient memory layout due to column major default in Eigen.